CsharpExtrasTest/Extensions/ArrayExtensionTest.cs

```diff
@@ -169,33 +169,29 @@ public void TestGivenStringArrayWhenGetSubArrayWithInvalidValuesThenFullArrayRet
         //non-mvp: Add test for getting subarray from out of bounds on only one side. 
         // For example: -1 to the middle of the source array
 
-        //non-mvp: Consider changing test to take a tupple of input array and number of expected blanks
-        // This will probably be a better test since the test uses the same code as the implementation
-
-        private static IEnumerable<string[]> StringArrayProviderForRemoveBlanksTesting()
+        private static IEnumerable<(string[] Result, string[] ExpectedBlanks)> TupleArrayProviderForRemoveBlanksTesting()
         {
-            return new List<string[]> {
-                new string[] { "a1", "", "a3", "a4", "  " },
-                new string[] {  },
-                new string[] { "a1" },
-                new string[] { "" },
-                new string[] { "  ", "a2" },
+            return new List<(string[] input, string[] expected)>
+            {
+                (input: new string[] {"a1", "", "a3", "a4", "  "}, expected: new string[] {"a1", "a3", "a4"}),
+                (input: new string[] { }, expected: new string[] {}),
+                (input: new string[] {"a1"}, expected: new string[] {"a1"}),
+                (input: new string[] {""}, expected: new string[] {}),
+                (input: new string[] {"  ", "a2"}, expected: new string[] {"a2"}),
             };
         }
 
         [Test]
         [Category("Unit")]
-        [TestCaseSource("StringArrayProviderForRemoveBlanksTesting")]
-        public void TestGivenStringArrayWhenRemoveBlankEntriesCalledThenAllBlankEntriesAreRemoved(string[] arr)
+        [TestCaseSource("TupleArrayProviderForRemoveBlanksTesting")]
+        public void TestGivenStringArrayWhenRemoveBlankEntriesCalledThenAllBlankEntriesAreRemoved((string[] input, string[] expected) arr)

```

Comment:
Change the variable name `arr` to `tuple` (or something that makes it clear that it's no longer an array)

Reply:
I changed the name of the test  but totally forgot it about the variable name! Thank you!